### ISSUE TITLE
Failed Test PR #294 - filled trade not logged properly causing an error when using amm_strategy

### ISSUE THREAD

**Issue by PtrckM** [CoinAlpha/hummingbot#304]
**Describe the bug**
- An error observe when running an amm strategy with `concurrent_orders_submission` enabled.  During the filled trade event, the injective side does not show filled order on the left pane and error out the filled order on the right pane. However checking the csv and sqlite file the filled trade was recorded. Crosschecking on the exchange the said order was confirmed filled. Performing a `history` command will also throw an error.

![image](https://user-images.githubusercontent.com/54516858/214804046-d0ca05c4-6a6e-47e0-9f42-5606e585308c.png)

```
2023-01-26 17:36:12,950 - 68014 - hummingbot.client.hummingbot_application - ERROR - 'NoneType' object has no attribute 'creation_timestamp'
Traceback (most recent call last):
  File "/Users/patrick/Documents/HB/injective_oss/hummingbot/client/hummingbot_application.py", line 240, in _handle_command
    f(**kwargs)
  File "/Users/patrick/Documents/HB/injective_oss/hummingbot/client/command/history_command.py", line 52, in history
    self.list_trades(start_time)
  File "/Users/patrick/Documents/HB/injective_oss/hummingbot/client/command/history_command.py", line 228, in list_trades
    df: pd.DataFrame = TradeFill.to_pandas(queried_trades)
  File "/Users/patrick/Documents/HB/injective_oss/hummingbot/model/trade_fill.py", line 124, in to_pandas
    age = pd.Timestamp(int(trade.timestamp / 1e3 - trade.order.creation_timestamp / 1e3),
AttributeError: 'NoneType' object has no attribute 'creation_timestamp'
```
![image](https://user-images.githubusercontent.com/54516858/214805524-980cf9db-a0bb-4794-8789-3f4c46b9ccf0.png)
![image](https://user-images.githubusercontent.com/54516858/214805576-6abfe776-135b-4e93-a08f-f337e6fc25ce.png)

** Test Result Comparison**
1. Clean clone of branch the install and compile it.
2. Start the client and gateway
3. create a amm strategy using injective and one centralized (binance)
4. set `concurrent_orders_submission` to true and adjust minimum profitability then start strategy
5. when filled trade event occurs on the injective side it does not show filled trade however it recorded it on the csv/sqlite file causing an error when `history` command was executed.

** Steps To Reproduce**
- new connector to OSS

**Attachments**
[injective_oss.zip](https://github.com/CoinAlpha/hummingbot/files/10508081/injective_oss.zip)

**Comment by vic-en** [comment]
@petioptrv  can you look at this when you get a chance? Thanks!

**Comment by petioptrv** [comment]
Should be addressed by [this commit](https://github.com/CoinAlpha/hummingbot/pull/294/commits/4cd8577bd1ad5b249b66ea46513ddac898b31d6b)

**Comment by PtrckM** [comment]
> Should be addressed by [this commit](https://github.com/CoinAlpha/hummingbot/pull/294/commits/4cd8577bd1ad5b249b66ea46513ddac898b31d6b)

Unable to retest due to blocker https://github.com/CoinAlpha/hummingbot/issues/307

**Comment by petioptrv** [comment]
I have added a fix for #307

**Comment by RC-13** [comment]
Re-test this behavior and so far did not see the error mentioned in the ticket. Number of filled trades record match as well in the client and exchange. I'm going to close this issue for now, and re-open if seen again.
